packages/types/src/utils/models/ProviderSecrets.ts

```diff
@@ -104,6 +136,9 @@ export interface OVirtProviderSecret {
   /**
    * OVirt server cacerts, can be a linked list of multple certifications.
    *
+   * NOTE: cacert is not optional because
```

Comment:
@bkhizgiy hi, please keep me honest here

Reply:
update: ovirts image-io has a bug that privent fatching images using insecut flag, @bkhizgiy posted a PR fixing it to ovirt, so,

currently because of the bug in ovirt, it's not optional yet, once the fix will be merged it will be optional

Reply:
changed the field to optional, left the comment explaining we need to wait for a bug fix in ovirt to actually using it as an optional field.

Reply:
Once this PR is merged should be working for ovirt as well.
https://github.com/oVirt/ovirt-imageio/pull/176